release_notes_generator/utils/utils.py

```diff
@@ -57,19 +57,3 @@ def get_change_url(
         changelog_url = f"https://github.com/{repo.full_name}/compare/{rls.tag_name}...{tag_name}"
 
     return changelog_url
-
-
-def detect_row_format_invalid_keywords(row_format: str, row_type: str = "Issue") -> list[str]:
```

Comment:
Check the imports for utils.

Reply:
Cleanup done in commit 3a77a9ec12c250ead4ad816666a634511d384a2e.